engines/director/lingo/lingo-object.cpp

```diff
@@ -208,6 +208,8 @@ Common::String Lingo::normalizeXLibName(Common::String name) {
 		if (name.hasSuffixIgnoreCase(".xlib"))
 			name = name.substr(0, name.size() - 5);
 	} else if (platform == Common::kPlatformWindows) {
+		int pos = name.findLastOf("\\");
```

Comment:
This should be `size_t` to match the prototype of the `findLastOf`. Also, you need to check it against `Common::npos` when the separator is not found. You better not assume that `npos` currently equals `-1`, e.g. `pos + 1`

Reply:
Fixed